.circleci/continue_config.yml

```diff
@@ -2927,6 +2927,20 @@ jobs:
             export TEST_PHP_EXECUTABLE=$(which php)
             php run-tests.php -d "extension=/mnt/ramdisk/cargo/release/libdatadog_php_profiling.so" --show-diff -g "FAIL,XFAIL,BORK,WARN,LEAK,XLEAK,SKIP" "phpt"
 
+      - run:
+          name: clippy NTS (for select platforms)
+          command: |
+            if [[ "x86_64-unknown-linux-gnu" != "<< parameters.triplet >>" ]] ; then
```

Comment:
Do we need to run clippy for all PHP versions, or would a single compilation be enough?

Reply:
There are PHP version differences, although mostly they are PHP 7 vs PHP 8.